README.md

```diff
@@ -1,14 +1,18 @@
 # checkout-3ds-sdk-ios-framework
 
-Releases for CheckoutEventLoggerKit
+Releases for `CheckoutEventLoggerKit`
+
+`CheckoutEventLoggerKit` is an internal tool for securely logging impersonal usage data in SDKs.
+
+This binary is not a Checkout.com product, and will receive no support.
```

Comment:
This would be too extreme, lets get some view from Kevin once he is back next week, For now delete this line 7.